requirements/mminstall.txt

```diff
@@ -0,0 +1,2 @@
+mmcv-full 1.2.4 1.4.0
+mmdet 2.10.0 3.0.0
```

Comment:
mmdet 2.10.0 2.11.0

Reply:
Thanks for your advice. I have updated the version of mmdet in mminstall.txt.